.travis.yml

```diff
@@ -12,5 +12,17 @@ php:
   - 7.1
   - hhvm
 
+matrix:
+  include:
+    - php: 7
+      env: PHPSTAN=1
+    - php: 7.1
+      env: PHPSTAN=1
```

Comment:
I am used to putting it separately in case you'd need to add it allow_failures temporarily.